### ISSUE TITLE
Make revisions more visual and cohesive across site wide changes

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#52138]
As part of the FSE Outreach Program's [latest call for testing](https://make.wordpress.org/test/2023/06/07/fse-program-testing-call-24-momery-makeover/#), the template and template part revisions were explored. While not making it into 6.3, it felt important to note the success of the visual approach of Styles and the desire to have a visual experience with templates and template parts:

>This option works fine, but the interface is obsolete. After I saw to how work the revisions in styles, it’s ideal option to these revisions. It’s difficult to see which changes are applying or no.

>I feel uncomfortable to roll-back template changes using the provided revision states located at the bottom of a particular template tab. The reason for this discomfort is that the revision states appear to be too raw. https://s.id/1NoxK Since we rarely used the Code editor in building a site/pages in the Site Editor, seeing loads of HTML block codes as reference points to roll back to feels unfamiliar within our current workflow.

This ties in with the [following phase 3 project area:](https://make.wordpress.org/core/2023/03/24/phase-3-collaboration/)

>Post revisions interface. Make them more visual, aware of individual blocks, and explore adding the ability to schedule revisions across multiple parts of a site. For example, being able to target updates for an event or campaign that might require coordinating the scheduling of multiple content types and resources. This also overlaps with addressing theme switching flows and scheduling that leverage the flexibility of block templates and styles.

Also on my mind with this effort is when to use each revision system as that's already coming up as a point of confusion as seen in [this piece of feedback from a different call for testing: ](https://make.wordpress.org/test/2023/05/18/fse-program-testing-call-23-rapid-revamp/#comment-3059)

>Saved and clicked to go into revisions. I did not have an option to remove the block adjustment. Looking at the revisions preview it looked like the block was the same independent of the revision system. I expected to go back to what it looked like before I adjusted the drag handles.

I didn't see an issue for this just yet in the repo (but found this trac ticket https://core.trac.wordpress.org/ticket/48574) so opening for now as a place to gather additional feedback, designs, ideas, etc.  as I didn't want to lose this feedback as it's coming in cc @WordPress/gutenberg-design

**Comment by mtias** [comment]
Yeah, all of these would be a direct part of the work of improving the interface of block revisions generally once we scope that out

**Comment by jasmussen** [comment]
This appears to be resolved by the Visual Revisions work that has since landed on trunk (packages/editor/src/components/post-revisions-timeline, post-revisions-preview, revisions-code-diff, revisions-canvas, plus shareable-URL support in [#79934](https://github.com/WordPress/gutenberg/issues/79934)/[#81205](https://github.com/WordPress/gutenberg/issues/81205) and a code diff view in [#80314](https://github.com/WordPress/gutenberg/issues/80314)). This brings the visual, timeline-based revisions experience, the same approach requested here, modeled on Styles revisions, to templates and template parts as well, with e2e coverage added in [#76923](https://github.com/WordPress/gutenberg/issues/76923). Closing as fixed; please reopen with a specific gap if the current Visual Revisions UI still falls short for a particular site-wide-changes scenario.
